service/snap/snap.go

```diff
@@ -91,9 +91,13 @@ type Installable interface {
 	// Name returns the name of the application
 	Name() string
 
-	// Install returns a way to install one application with all it's settings.
```

Comment:
Yes agree. Something like `InstallCmd` and `AcknowledgeAssertsCmd` would be much clearer here

Reply:
I think `InstallArgs` and `AcknowledgeAssertsArgs` (resp.) are the clearest, because they don't return the full command, just the part of the command which comes after the initial `snap` command